client/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
-import App from './components/App';
+import App from './components/App/index';
```

Comment:
you don't have to add `index` at the end of the route, it will be taken explicitly by React 
```suggestion
import App from './components/App';
```